DeploymentCloud/Deployment.Common/Helpers/UtilityModule.psm1

```diff
@@ -483,7 +483,7 @@ function Get-CosmosDBConnectionString([string]$Name) {
     -ResourceType "Microsoft.DocumentDb/databaseAccounts" `
     -ApiVersion "2015-04-08" `
     -resourceGroupName $resourceGroupName `
-    -Name $docDBName `
+    -Name $Name `
```

Comment:
Any reason why we need this change?

Reply:
This was a bug. The parameter being passed to the function was Name but we were using docDBName instead of Name